.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   build:
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
I don't think we want to fail fast any more, since we test different packages in different parts of the build matrix.